README.md

```diff
@@ -1,83 +1,88 @@
+| SSAS_ADMIN_PORT <br/> SSAS_PUBLIC_PORT <br/> SSAS_HTTP_TO_HTTPS_PORT |    No    |  X   |  X   | These values are not yet used by code. Intended to allow changing port assignments. If used, will affect BCDA SSAS URL vars.                                                                                        |
+| SSAS_ADMIN_SIGNING_KEY <br/>or<br/> SSAS_ADMIN_SIGNING_KEY_PATH      |   Yes    |  X   |      | Provides the plaintext/path of the admin server signing key. When setting vars for AWS envs, you must include a var for the key material. Set either the key or the path but not both.                              |
+| SSAS_PUBLIC_SIGNING_KEY <br/>or <br/>SSAS_PUBLIC_SIGNING_KEY_PATH    |   Yes    |  X   |      | Provides the plaintext/path of the public server signing key. When setting vars for AWS envs, you must include a var for the key material. Set either the key or the path but not both.                             |
+| SSAS*TOKEN_BLACKLIST_CACHE* <br/> CLEANUP_MINUTES                    |    No    |  X   |      | Tunes the frequency that expired entries are cleared from the token blacklist cache. Defaults to 15 minutes.                                                                                                        |
+| SSAS*TOKEN_BLACKLIST_CACHE* <br/> TIMEOUT_MINUTES                    |    No    |  X   |      | Sets the lifetime of token blacklist cache entries. Defaults to 24 hours.                                                                                                                                           |
+| SSAS*TOKEN_BLACKLIST_CACHE* <br/> REFRESH_MINUTES                    |    No    |  X   |      | Configures the number of minutes between times the token blacklist cache is refreshed from the database.                                                                                                            |
+| BCDA_TLS_CERT                                                        | Depends  |  X   |      | The cert used when the SSAS service is running in secure mode. When setting vars for AWS envs, you must include a var for the cert material. This var should be renamed to SSAS_TLS_CERT.                           |
+| BCDA_TLS_KEY                                                         | Depends  |  X   |      | The private key used when the SSAS service is running in secure mode. When setting vars for AWS envs, you must include a var for the key material. This var should be renamed to SSAS_TLS_KEY.                      |
+| SSAS_CLIENT_ASSERTION_AUD                                            |   Yes    |  X   |      | The audience (aud) claim value required when authenticating using client assertion tokens (v2/token).                                                                                                               |
 
```

Comment:
I recently came across these environment variables that haven't been documented. It may be worth adding to the list

```
	timeoutDays := cfg.GetEnvInt("SSAS_CRED_TIMEOUT_DAYS", 60)
	expirationDays := cfg.GetEnvInt("SSAS_CRED_EXPIRATION_DAYS", 90)
	warningDays := cfg.GetEnvInt("SSAS_CRED_WARNING_DAYS", 7)
```

Reply:
added